awx/main/tasks.py

```diff
@@ -822,41 +830,57 @@ def build_env(self, instance, **kwargs):
         # Update PYTHONPATH to use local site-packages.
         # NOTE:
         # Derived class should call add_ansible_venv() or add_awx_venv()
-        if self.should_use_proot(instance, **kwargs):
+        if self.should_use_proot(instance):
             env['PROOT_TMP_DIR'] = settings.AWX_PROOT_BASE_PATH
-        env['AWX_PRIVATE_DATA_DIR'] = kwargs['private_data_dir']
+        env['AWX_PRIVATE_DATA_DIR'] = private_data_dir
         return env
 
-    def should_use_proot(self, instance, **kwargs):
+    def should_use_proot(self, instance):
         '''
         Return whether this task should use proot.
         '''
         return False
 
-    def build_inventory(self, instance, **kwargs):
+    def build_inventory(self, instance, private_data_dir):
```

Comment:
I've been going through these, and all of these line up with what I was expecting, and using within the inventory source injector stuff. So that's good.